package.json

```diff
@@ -1,26 +1,46 @@
 {
   "name": "extension-skeleton",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "description": "Bare Google Chrome extension skeleton.",
-  "dependencies": {},
-  "devDependencies": {
-    "browserify": "~7.0.3",
-    "eventemitter2": "^0.4.14",
-    "grunt": "~0.4.5",
-    "grunt-browserify": "~3.2.1",
-    "grunt-contrib-clean": "~0.6.0",
-    "grunt-contrib-copy": "~0.7.0",
-    "grunt-contrib-jshint": "~0.10.0",
-    "grunt-contrib-uglify": "~0.6.0",
-    "grunt-contrib-watch": "~0.6.1",
-    "grunt-crx": "^1.0.5",
-    "grunt-mkdir": "~0.1.2",
-    "grunt-mocha-test": "~0.12.4",
-    "mocha": "^1.20.0",
-    "sinon": "~1.12.2"
-  },
   "engines": {
-    "node": ">= 0.10.0"
+    "node": "7.0.0",
+    "npm": "3.10.8"
+  },
+  "scripts": {
+    "build": "mkdir -p ./build/prod && webpack --config ./webpack/webpack.config.prod.js",
+    "start": "mkdir -p ./build/dev && webpack --config ./webpack/webpack.config.dev.js",
+    "lint": "eslint src/js/",
+    "deploy": "npm run build && now ./build",
```

Comment:
This will not work, I guess.
1: there is no need to deploy anything anywhere,
2: now.sh is not listed among dependencies and README doesn't say it should be installed globally either.
---> Please remove.